angular/src/lib/popover/examples/popover-default.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, ViewChild, TemplateRef } from '@angular/core';
+
+@Component({
+  selector: 'example-popover-default',
+  template: `
+  <md-badge mdPopover="Hello World!">Popover with text only</md-badge>
+  <br>
+  <md-badge color="blue" [mdPopover]="tootltipTemplate">Popover from template</md-badge>
+      <ng-template #tootltipTemplate>
+          <md-badge color="red">HOWDY</md-badge>
+          <div class="box">THis is cool</div>
+      </ng-template>
+  `,
+  styles: []
+})
+export class PopoverDefaultComponent {
+  @ViewChild('template') popoverTemplate: TemplateRef<any>;
```

Comment:
Doesn't look like this ViewChild is being used, I do see a reference called #tooltipTemplate though.

Reply:
thanks!  I'll take it out.